classes/scripts/runner.php

```diff
@@ -608,7 +606,7 @@ public static function enableAutorun($name)
 						{
 							$autorunner->writeError($exception->getMessage());
 
-							exit($exception->getCode());
+							exit(2);
```

Comment:
Should we use exception code when it's available or always return `2` ?

Thing to know is an exception can have a code equal to `3` which will get confusing (is it an error or an exception?)...